com.adobe.Flash-Player-Projector.json

```diff
@@ -14,13 +14,17 @@
         "--socket=pulseaudio",
         /* Get access to the files */
         "--filesystem=host:ro",
-        "--filesystem=xdg-run/gvfs:ro"
+        "--filesystem=xdg-run/gvfs:ro",
+        /* Cache directory */
+        "--filesystem=~/.local/share/com.adobe.Flash-Player-Projector/.adobe:create",
+        "--filesystem=~/.local/share/com.adobe.Flash-Player-Projector/.macromedia:create"
```

Comment:
If you are going though this trouble why not put it where everything else goes? `~/.var/app/com.adobe.Flash-Player-Projector`

Reply:
Good idea. Still blocking on flashplayer not crashing on exit though...